### PR TITLE
fix counting liars

### LINKED ISSUE #4134: Contact Form Submission - Minor Mistake (Counting Liars Sol)
Someone submitted the contact form!

**URL**: https://usaco.guide/problems/usaco-1228-counting-liars/solution
**Module**: Counting Liars Sol
**Topic**: Minor Mistake
**Message**: 
the solution code is wrong: https://forum.usaco.guide/t/potentially-wrong-solution-on-usaco-guide/5322

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [x] I have linked this PR to any issues that it closes.

closes #4134

### CODE REVIEW

File: solutions/bronze/usaco-1228.mdx [hunk 1]
```diff
@@ -41,16 +41,14 @@ int main() {
...
 	for (int i = 1; i < n; i++) {
-		// Add up all the cows that are already lying to the left of our pos.
+		// Add up all the cows that are lying to the left of our pos.
 		lying_left[i] += lying_left[i - 1];
 
-		if (cows[i - 1].second == 'L' && cows[i].first > cows[i - 1].first) {
+		if (cows[i - 1].second == 'L' && cows[i].first >= cows[i - 1].first) {
 			/*
 			 * If the cow before says our position is to the left
 			 * but their position is strictly less than our position,
```
Comment (envyaims) [inline solutions/bronze/usaco-1228.mdx:54]:
```suggestion
			 * but their position is strictly less than or equal to our position,
```

File: solutions/bronze/usaco-1228.mdx [hunk 2]
```diff
@@ -180,35 +179,28 @@ for _ in range(n):
...
 
-	if cows[i - 1].statement == "L" and cows[i].pos > cows[i - 1].pos:
-		"""
-		If the cow before says our position is to the left
-		but their position is strictly less than our position,
-		they're lying.
-		"""
+	if cows[i - 1].statement == "L" and cows[i].pos >= cows[i - 1].pos:
+		# If the cow before says our position is to the left
+		# but their position is strictly less than our position, they're lying.
```
Comment (envyaims) [inline solutions/bronze/usaco-1228.mdx:193]:
```suggestion
		# but their position is strictly less than or equal to our position, they're lying.
```

**Review by envyaims — COMMENTED** [review]
moo

**Review by envyaims — APPROVED** [review]

File: solutions/bronze/usaco-1228.mdx [hunk 3]
```diff
@@ -41,35 +41,31 @@ int main() {
...
-	/*
-	 * lying_left[i] stores the number of cows to the left of cow i
-	 * that must be lying given that Bessie is at the position of cow i.
-	 */
+	// lying_left[i] stores the number of cows to the left of cow i
+	// that must be lying given that Bessie is at the position of cow i.
 	vector<int> lying_left(n);
 	for (int i = 1; i < n; i++) {
-		// Add up all the cows that are already lying to the left of our pos.
+		// Add up all the cows that are lying to the left of our pos.
```
Comment (ryanchou-dev) [inline solutions/bronze/usaco-1228.mdx:48]:
```suggestion
		// Add up all the cows that are lying to the left of our position.
```

**Review by ryanchou-dev — APPROVED** [review]

**Pull request merged**